test/verify/check-system-info

```diff
@@ -525,23 +531,26 @@ class TestSystemInfo(MachineCase):
         spoof_threads(2, True, True, 'param1 mitigations=nosmt,something param3=value3')
         spoof_threads(2, True, False, 'param1 mitigations=something param3=value3')
         spoof_threads(2, False, cmdline='param1 nosmt=someunknown param3=value3')
-        # Most OSes don't set nosmt by default, but there are some exceptions
-        expect_smt_default = m.image in ["fedora-coreos"]
-        spoof_threads(2, True, expect_smt_default, None)
+        spoof_threads(2, True, self.expect_smt_default, None)
 
-        # Switch nosmt option
-        b.reload()
-        b.enter_page('/system/hwinfo')
-        b.click('#hwinfo button:contains(Mitigations)')
+    def testCPUSecurityMitigationsEnable(self):
+        b = self.browser
+        m = self.machine
 
+        # spoof SMT
+        m.write('/usr/local/bin/lscpu', lscpu.format(2))
```

Comment:
This file is left behind. Should not do any harm though.

Reply:
The test does `self.restore_dir("/usr/local/bin")`, thus cleans this up. I found it more convenient to do the bind-mount restoration of the w

Reply:
Ah, this test is destructive anyway, please ignore :)